src/pages/product.js

```diff
@@ -7,6 +7,9 @@ import withShop from "containers/shop/withShop";
 import Layout from "components/Layout";
 import ProductDetail from "components/ProductDetail";
 
+// TODO: Data will eventually come from GraphQL
+import sampleData from "components/ProductDetail/sampleData";
```

Comment:
What are your thoughts on using a jest convention for naming mocked data. I like using a folder named `__mocks` that would contained a file name `product.js` with the product object? and we can use those mocks in jest tests.